website/public/main.css

```
@@ -718,7 +721,48 @@ kbd.keystroke {
     bottom: 0;
   }
 }
-
+.comparison-mobile {
+  display: none;
+}
+#editors-support {
+  display: none;
+}
+@media only screen and (max-width: 684px) {
+  .comparison {
+    display: none;
+  }
+  .comparison-mobile {
+    display: initial;
+  }
+  .comparison-mobile table {
+    width: 100%;
+    text-align: center;
+    border-collapse: collapse;
+    border: 1px solid gray;
+  }
+  .comparison-mobile table th,
+  td {
```

Must fix: Do not apply rules to all `td` elements. I think you meant this
```suggestion
  .comparison-mobile table td {
```